SQL_SELECT_HW

crisforp opened the pull request:
@mozuela

crisforp committed:
Task: writing queries with multiple solutions

crisforp committed:
Add files via upload

SQL_SELECT_Cristian_Luca_HW.sql updated

mozuela commented:
Hi Cristian, 
Schema is missing in the queries
UPPER or LOWER is missing when using category name column 
As a good practice try to incorporate the comments into the code 
Keep UPPER for all the SQL commands 
Part 1
1) - release year and rental rate columns are not needed in the results, remember each extra column/duplicate data consume resources and impacts performance
- alias in the second join don't match the condition alias, was the code execute without issues? 
2) Better to group by store id , Join type needs to be explicitly mentioned ex: INNER/LEFT etc
4) COALESCE is not needed, since case/SUM will produce 1 or 0
Part 2
1) Join type needs to be explicitly mentioned 
2) Join type needs to be explicitly mentioned, rating description is incomplete 
'G' -> 'All Ages'
'PG-13'-> 'Inappropriate for Children Under 13'
'R'->'Children Under 17 Require Accompanying Adult'
'NC-17'->'Inappropriate for Children Under 17'
'PG'-> 'Parental Guidance Suggested'

crisforp committed:
Add files via upload

I fixed the code with minor changes

crisforp commented:
I fix the code with minor changes.

mozuela commented:
Hi Cristian, the only thing missing is when category.name when used in CASE, please update to UPPER and merge

crisforp committed:
Add files via upload

Hi Cristian, the only thing missing is when category.name when used in CASE, please update to UPPER and merge
